File: 0075-sort-colors/0075-sort-colors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int n=nums.size();
        for(int i=1; i<n; i++){
            int j=i;
            while(j>0){
                if(nums[j] < nums[j-1]){
                    swap(nums[j],nums[j-1]);
                }else{
                    break;
                }
                j--;
            }
        }
    }
};

File: 0075-sort-colors/0075-sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        color = {0: 0, 1: 0, 2: 0}
        
        for n in nums:
            color[n] += 1
        
        ind =0
        for c in color:
            for col in range(color[c]):
                nums[ind] = c
                ind += 1
            
            
        
        